tests/test_data.py

```diff
@@ -113,7 +152,7 @@ def test_v3_sensor_parsing():
         back_module_data = {
             "sensors": [
                 {"type": "co2", "value": 35},
-                {"type": "reedVelocity", "value": 20.1},
+                {"type": "reed_velocity", "value": 20.1},
```

Comment:
This was correct before. V3 mqtt data uses camel case, but in raspicam for legacy reasons we still store the data in snake case. This is imitating MQTT data, so it should be camel case.